localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -466,6 +478,8 @@ void EKFLocalizer::measurementUpdateTwist(
       "twist frame_id must be base_link");
   }
 
+  pitch_rate_ = twist.twist.twist.angular.y;
```

Comment:
Why is this here in the first place? :thinking: 

Why not just extracting `twist.twist.twist.angular.y` from `twist_queue_` in e.g. line 207? 

Generally speaking, it is better to avoid using member variables, and this pitch_rate_ member variable seems not necessary.

Reply:
OK.
But I don't know how to extract `twist.twist.twist.angular.y` from t`wist_queue_` in below code unit.
```c++
    // save the initial size because the queue size can change in the loop
    const size_t n = twist_queue_.size();
    for (size_t i = 0; i < n; ++i) {
      const auto twist = twist_queue_.pop_increment_age();
      measurementUpdateTwist(*twist);
    }
    DEBUG_INFO(get_logger(), "[EKF] measurementUpdateTwist calc time = %f [ms]", stop_watch_.toc());
    DEBUG_INFO(get_logger(), "------------------------- end Twist -------------------------\n");
  }
``` 
For example, `twist_queue_[0].twist.twist.angular.y`?